src/sssom/util.py

```diff
@@ -294,10 +295,11 @@ def _standardize_curie_or_iri(curie_or_iri: str, *, converter: Converter) -> str
         - If the string represents a CURIE, tries to standardize it. If not possible, returns the original value
         - Otherwise, return the original value
     """
-    if is_iri(curie_or_iri):
-        return converter.standardize_uri(curie_or_iri) or curie_or_iri
-    if is_curie(curie_or_iri):
-        return converter.standardize_curie(curie_or_iri) or curie_or_iri
+    if converter.is_uri(curie_or_iri):
+        # TODO switch to compress
+        return converter.standardize_uri(curie_or_iri, strict=True)
```

Comment:
Shouldn't it be strict=False to replicate the original functionality?

Reply:
if it already passes the `converter.is_uri` check, then it can indeed be standardized. The strict flag makes the typing work better.

Reply:
Duuuuuh. Sorry you are right!